ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -152,6 +153,27 @@
         (get-unique-native-id context subscription))
       native-id)))
 
+(defn check-subscriber-id
```

Comment:
Can this be renamed to be more descriptive to what it's doing? Something like `add-subscriber-id-if-missing`

Reply:
Agreed, replacement functions are more descriptively named. Actually, they use your suggested name exactly